integreat_cms/cms/views/organizations/organization_list_view.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...models import Organization
+from .organization_content_mixin import OrganizationContextMixin
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.http import HttpRequest, HttpResponse
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.view_organization"), name="dispatch")
+class OrganizationListView(TemplateView, OrganizationContextMixin):
+    """
+    View for listing organizations (either non-archived or archived organizations depending on
+    :attr:`~integreat_cms.cms.views.organizations.organization_list_view.OrganizationListView.archived`)
+    """
+
+    #: Template for list of non-archived organizations
+    template = "organizations/organization_list.html"
+    #: Template for list of archived organizations
+    template_archived = "organizations/organization_list_archived.html"
+    #: Whether or not to show archived organizations
+    archived = False
+
+    @property
+    def template_name(self) -> str:
+        """
+        Select correct HTML template, depending on :attr:`~integreat_cms.cms.views.organizations.organization_list_view.OrganizationListView.archived` flag
+        (see :class:`~django.views.generic.base.TemplateResponseMixin`)
+
+        :return: Path to HTML template
+        """
+        return self.template_archived if self.archived else self.template
+
+    def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
```

Comment:
I think this is missing the paginator. Given that there are typically not very many organizations, this is probably fine, but then we should the pagination options box at the html list view.